bee-ledger/Cargo.toml

```diff
@@ -54,3 +54,4 @@ workers = [
   "tokio-stream",
   "url/serde"
 ]
+cpt2 = [ "bee-message/cpt2" ]
```

Comment:
Actually I wouldn't even add this for now

Reply:
Do you mean always assume that feature is enabled in bee? Because removing that will break building with that feature enabled as the ledger uses those extra gated variants.

Reply:
I meant don't even consider cpt2 types in the ledger at all

Reply:
You don't think they'll ever be needed there? Might be better to plan ahead a little.

Reply:
At the moment no, I'd start clean and add if really necessary tbh

Reply:
How about the API?

Reply:
Hmm good question. I would say the same, the least amount of logic the better. Maybe hold on to that until tomorrow, I need to sleep on it.

Reply:
Only chronicle seems to have the need for the old types so atm I don't think the node (ledger + api) should care about that. But yeah, need to think about it a bit more and see with the team.

Reply:
👍